<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Заполнение архива</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>игрушки
с треем</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_go">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_success">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>380</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_complaints">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>380</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>130</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Найти новые ПП</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_newPrograms">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>211</width>
        <height>380</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>300</width>
        <height>250</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_getNewPrograms">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Получить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_amountNewElements">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>28</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_fails">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>302</y>
        <width>300</width>
        <height>125</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_failsElements">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>286</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Фэйлы тут</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_currentSeries">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>265</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clearNewProgramsTextBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>25</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_resultHere">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>3</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Результат тут</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_amountNewElementsText">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>28</y>
        <width>110</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clearNewProgramsFailTextBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>277</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Настройки системы</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_savePaths">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>520</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_from2k">
         <property name="text">
          <string>Откуда</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_from2k"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_todocs2k">
         <property name="text">
          <string>Куда
оформления</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_todocs2k"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_toprogs2k">
         <property name="text">
          <string>Куда
программы</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_toprogs2k"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_excel2k">
         <property name="text">
          <string>БД F2K.xls</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_excel2k"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
